wls-gui-wahllokalsystem/src/composables/tasks/taskFactories/kopfdatenTaskFactory.ts

```diff
@@ -0,0 +1,28 @@
+import type { ExtendedWahlMetaData } from "@/composables/tasks/ExtendedWahlMetaData.ts";
+import type { TaskFactoryContext } from "@/composables/tasks/TaskFactoryContext.ts";
+import type { TaskFactoryInterface } from "@/composables/tasks/TaskFactoryInterface.ts";
+import type { Task } from "@/types/tasks/Task.ts";
+
+import { useKopfdatenStore } from "@/stores/kopfdatenStore.ts";
+
+export function useKopfdatenTaskFactory(): TaskFactoryInterface {
+  function createTasks(taskFactoryContext: TaskFactoryContext): Task[] {
+    return taskFactoryContext.extendedWahlMetaData.map(createTask);
+  }
+
+  function createTask(taskFactoryMetaData: ExtendedWahlMetaData): Task {
+    const { loadKopfdaten } = useKopfdatenStore();
+    return {
+      callback: () =>
+        loadKopfdaten(
+          taskFactoryMetaData.wahlID,
+          taskFactoryMetaData.wahlbezirkID
+        ),
+      name: "Kopfdaten - " + taskFactoryMetaData.wahlName,
```

Comment:
Ich finde es so lesbarer

```suggestion
      name: `Kopfdaten - ${taskFactoryMetaData.wahlName}`,
```